.env.example

```
@@ -146,6 +146,27 @@ SIWE_ALLOWED_CHAIN_IDS=1
 # Generate with: openssl rand -hex 32
 # Treat as a secret; rotate via deploy if compromised (invalidates in-flight nonces, 5-min TTL absorbs).
 NONCE_HMAC_SECRET=
+# REQUIRED — HMAC secret keying account-deletion barrier hashes and pseudonymous
+# deletion-record refs. Must be >= 64 hex chars (32 bytes).
+# Generate with: openssl rand -hex 32
+# PERMANENT: never rotate — rotation orphans every DeletedIdentity barrier row
+# (silently lifting the deletion bar) and breaks deletion-record lookups.
+DELETION_HASH_SECRET=
+# Rollout gate for DELETE /v2/accounts/me — fail-closed, only the exact
+# string "true" enables it (flip = infra PR + task-definition roll).
+ACCOUNT_DELETION_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ACCOUNT_DELETION_ENABLED key should go before the DELETION_HASH_SECRET key

(UnorderedKey)